label_studio_ml/examples/paddleocr/requirements.txt

```diff
@@ -0,0 +1,13 @@
+label-studio-ml
```

Comment:
it is better to pinpoint versions in requirements, to avoid failed dependencies in future docker image rebuilds

Reply:
done